3rdParty/velocypack/include/velocypack/Slice.h

```diff
@@ -692,7 +693,24 @@ class Slice {
 
     throw Exception(Exception::InvalidValueType, "Expecting type String");
   }
-  
+
+  std::string_view stringView() const {
```

Comment:
hold on why not replace StringRef ??

Reply:
just `using StringRef = std::string_view` and fix the compilation errors

Reply:
Yes. That is about what should be done. I decided to stick to the `try_emplace` first and not to get sidetracked. It is more a TODO. I am not sure if we should do a breaking API change, as the velocpack lib is in drivers and public available. But switching internally to `std::string_view` is definitely the way to go!

Reply:
Right now we should exclude all VelocyPack library files from these changes, as it is currently a separate library with separate requirements (and no C++17 build infrastructure).

Reply:
https://github.com/arangodb/velocypack/pull/49 - build infrastructure updated

Reply:
https://github.com/arangodb/velocypack/pull/54

Reply:
`stringView()` is now officially in VelockyPack upstream.